src/android/plugin/google/maps/MyPluginLayout.java

```diff
@@ -281,6 +285,50 @@ public void inValidate() {
     this.frontLayer.invalidate();
   }
 
+  public void setDragging() {
+    this.mapDragging = true;
+  }
+
+  public void clearDragging() {
```

Comment:
Maybe this method can be called `setDraggingEnded` to indicate more clearly its intent